pandas/_libs/tslib.pyx

```diff
@@ -855,17 +851,11 @@ cdef inline bint _parse_today_now(str val, int64_t* iresult, bint utc):
     # We delay this check for as long as possible
     # because it catches relatively rare cases
     if val == "now":
-        iresult[0] = Timestamp.utcnow().value
-        if not utc:
+        if utc:
+            iresult[0] = Timestamp.utcnow().value
+        else:
             # GH#18705 make sure to_datetime("now") matches Timestamp("now")
-            warnings.warn(
-                "The parsing of 'now' in pd.to_datetime without `utc=True` is "
-                "deprecated. In a future version, this will match Timestamp('now') "
-                "and Timestamp.now()",
-                FutureWarning,
-                stacklevel=find_stack_level(),
-            )
-
+            iresult[0] = Timestamp("now").value
```

Comment:
Looks a bit more performant to use `Timestamp.now` instead

```
In [3]: %timeit Timestamp.now()
2.04 µs ± 32.3 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [4]: %timeit Timestamp("now")
2.25 µs ± 29.1 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)
```

Reply:
looks like `np.datetime64("now", "ns").view("i8")` is 2x faster than Timestamp.now().value

Reply:
turns out Timestamp.now() was the way to go. updated.